bower.json

```diff
@@ -23,6 +23,7 @@
     "lodash": "^4.13.1",
     "ng-ckeditor": "formio/ng-ckeditor#master",
     "ng-dialog": "^0.6.1",
-    "ng-formio": "^2.0.1"
+    "ng-formio": "^2.0.1",
+    "font-awesome": "^4.6.3"
```

Comment:
The majority of the component definitions define a font-awesome icon. Would that make it a dependency?
https://github.com/assignar/ngFormBuilder/blob/f281e9d4cf7c09ccb9345af0f1c0e8c031bb43f0/src/components/textarea.js#L6